tools/scaninc/scaninc.h

```diff
@@ -1,4 +1,4 @@
-// Copyright(c) 2015-2017 YamaArashi
+// Copyright(c) 2019 Phlosioneer
```

Comment:
Don't remove Yama's copyright

Reply:
Oops, that was just supposed to be the new files.